app/src/main/java/ai/elimu/appstore/synchronization/AppListActivity.java

```diff
@@ -16,55 +16,53 @@
 import ai.elimu.appstore.R;
 import ai.elimu.appstore.dao.ApplicationDao;
 import ai.elimu.appstore.model.Application;
-import timber.log.Timber;
+import ai.elimu.appstore.util.Const;
 
 public class AppListActivity extends AppCompatActivity {
 
-    private TextView textViewLastSynchronization;
+    private TextView mTextViewLastSynchronization;
 
-    private List<Application> applications;
+    private List<Application> mApplications;
 
-    private ArrayAdapter arrayAdapter;
+    private ArrayAdapter mAdapterApps;
 
-    private ListView listViewApplications;
+    private ListView mListViewApps;
 
-    private ApplicationDao applicationDao;
+    private ApplicationDao mApplicationDao;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
-        Timber.i("onCreate");
+//        Timber.i("onCreate");
         super.onCreate(savedInstanceState);
-
         setContentView(R.layout.activity_app_list);
 
-        listViewApplications = findViewById(R.id.listViewApplications);
-
-        textViewLastSynchronization = findViewById(R.id.textViewLastSynchronization);
-
         BaseApplication baseApplication = (BaseApplication) getApplication();
-        applicationDao = baseApplication.getDaoSession().getApplicationDao();
+        mApplicationDao = baseApplication.getDaoSession().getApplicationDao();
+
+        initViews();
     }
 
-    @Override
-    protected void onStart() {
-        Timber.i("onStart");
-        super.onStart();
+    private void initViews() {
```

Comment:
Can you add some documentation explaining what this method does?

Reply:
The method initializes main views of this Activity.
Do I need to add that to the code?

Reply:
Just a few words summarizing what happens in the method. The more logic a method contains, the more important it is to add some documentation summarizing the functionality.